graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/fetchers/deployed/UpdateFetcher.java

```diff
@@ -200,4 +161,87 @@ protected Object get(DataFetchingEnvironment environment, StargateGraphqlContext
     }
     return modifiers;
   }
+
+  private AbstractBound<?> buildUpdateQuery(
+      EntityModel entityModel,
+      Collection<ValueModifier> modifiers,
+      List<BuiltCondition> whereConditions,
+      List<BuiltCondition> ifConditions,
+      Optional<Long> timestamp,
+      StargateGraphqlContext context) {
+    AbstractBound<?> query =
+        context
+            .getDataStore()
+            .queryBuilder()
+            .update(entityModel.getKeyspaceName(), entityModel.getCqlName())
+            .ttl(model.getTtl().orElse(null))
+            .timestamp(timestamp.orElse(null))
+            .value(modifiers)
+            .where(whereConditions)
+            .ifs(ifConditions)
+            .ifExists(model.ifExists())
+            .build()
+            .bind();
+    return query;
+  }
+
+  private void authorizeUpdate(
+      EntityModel entityModel, List<TypedKeyValue> primaryKey, StargateGraphqlContext context)
+      throws UnauthorizedException {
+    context
+        .getAuthorizationService()
+        .authorizeDataWrite(
+            context.getSubject(),
+            entityModel.getKeyspaceName(),
+            entityModel.getCqlName(),
+            primaryKey,
+            Scope.MODIFY,
+            SourceAPI.GRAPHQL);
+  }
+
+  private Function<List<MutationResult>, DataFetcherResult<Object>> getResultBuilder(
+      DataFetchingFieldSelectionSet selectionSet, DataFetchingEnvironment environment) {
+    return queryResults -> {
+      DataFetcherResult.Builder<Object> result = DataFetcherResult.newResult();
+      assert queryResults.size() == 1;
```

Comment:
Is it possible to share anything in here with the DeleteFetcher? Seems like there's a lot of similarities.

Reply:
Right, even more so after my changes for #1082. I'm adding a todo item to make those changes in that PR.